bot/cogs/moderation/infractions.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, bot: Bot):
 
     @command()
     async def ban(self, ctx: Context, user: FetchedMember, *, reason: str = None) -> None:
-        """Permanently ban a user for the given reason."""
+        """Permanently ban a user for the given reason.  Also removes them from the BigBrother watch list."""
```

Comment:
I think this reads better as a single sentence and keeps the imperative mood.

```suggestion
        """Permanently ban a user for the given reason and stop watching them with Big Brother."""
```